src/random.cpp

```diff
@@ -73,9 +73,15 @@ static constexpr uint32_t CPUID_F1_ECX_RDRAND = 0x40000000;
 static void RDRandInit()
 {
     //! When calling cpuid function #1, ecx register will have this set if RDRAND is available.
```

Comment:
While you're at it, can you move this comment down to after the #endif? It's intended to clarify the CPUID_F1_ECX_RDRAND constant.